backend/fms_core/viewsets/sample_next_step_by_study.py

```diff
@@ -62,6 +63,35 @@ def destroy(self, request, pk=None):
         else:
             return HttpResponseBadRequest(errors or f"Missing sample-next-step-by-study ID to delete.")
 
+    @action(detail=False, methods=["post"])
+    def destroy_list(self, request):
+        sample_ids = request.data.get("sample_ids", [])
+        study = request.data.get("study", None)
+        stepOrder = request.data.get("step_order", None)
+        if not sample_ids:
+            return HttpResponseBadRequest("No sample IDs provided.")
+        if not study:
+            return HttpResponseBadRequest("No study ID provided.")
+        if not stepOrder:
```

Comment:
be careful testing against falsy values ... 0 is false and is often a valid value ... test against the default value as much as possible instead (typically "is not None", but also [] and {}), Here step order start at 1, but this is never a bad thing to test for a more specific value.